lib/pharos/cluster_manager.rb

```diff
@@ -50,19 +41,18 @@ def load
       addon_dirs = addon_dirs.map { |dir| Pathname.new(dir).realpath.to_s }.uniq
 
       Pharos::AddonManager.load_addons(*addon_dirs)
-      Pharos::HostConfigManager.load_configs(@config)
     end
 
     def gather_facts
-      apply_phase(Phases::GatherFacts, config.hosts, ssh: true, parallel: true)
+      apply_phase(Phases::GatherFacts, config.hosts, parallel: true)
```

Comment:
Just thinking if `Phase` should provide info about parallelism?

Reply:
Done in #630 or other follow-up to this one